docs/guides/miscellaneous/how-to-write-a-how-to-guide.md

```diff
@@ -2,4 +2,214 @@
 title: How to write a how-to-guide
 ---
 
-This article is a stub.
+This guide shows how to create a new how-to guide in Great Expectations. By writing guides with consistent structure and styling, you can get your PRs approved faster and make the Great Expectations docs more discoverable, useful, and maintainable.
+
+# Background
+
+Documentation for Great Expectations is built using Docusaurus 2, a modern static website generator.  This allows for documentation to be written and built quickly, and and for the code that is displayed in the how-to-guides to be tested.  This means that whenever a code-snippet or configuration is displayed in the how-to-guide, it is actually part of a script that is fully integration tested.
+
+## Why test documentation?
+
+Testing provides guarantees that code or configuration displayed in the how-to-doc is fully integration tested with the most-current version of Great Expectations.  Since Great Expectations is a rapidly-changing codebase, there is a great deal of value making sure that the documentation is always up-to-date with the most current changes. Because the script is so central to writing the guides, we highly recommend that you **write the script first, before writing the how-to-guide**.
+
+## What does it mean for contributors?
+
+Typically, a PR for a how-to-guide will contain the following:
+
+1. The script(s) and integration test.
+    - The script will be located with the other integration tests in the `great-expectations/tests/integration/docusaurus` folder.
```

Comment:
Could you add a note here about making sure to mirror the directory structure of the how-to guide?